### ISSUE TITLE
GetAlbumUrl() no longer returns anything

### ISSUE THREAD

**Issue by erinxocon** [JohnnyCrazy/SpotifyAPI-NET#78]

**Comment by erinxocon** [comment]
pull request  #79 should fix this

**Comment by erinxocon** [comment]
The problem seemed to be that spotify changed the response that comes back when you grab https://open.spotify.com/album/<id> source code and look for the meta property that equals og:image.  It is now in a huge group of text with other properties.  I wrote the following code to handle it.  It should be a little more future proof since it will find it in a string of any length with any amount of forward slashes.  

```c#
string startString = "<meta property=\"og:image\"";
string endString = "\">";
foreach (string line in lines)
{
    if (line.Trim().Contains("<meta property=\"og:image\""))
    {
        int start = line.IndexOf(startString, 0) + startString.Length;
        int end = line.IndexOf(endString, start);
        string content = line.Substring(start, end - start);
        string[] l = content.Split(new[] { "/" }, StringSplitOptions.None);
        return "http://o.scdn.co/" + albumsize + @"/" + l[4].Replace("\"", "").Replace(">", "");
    }
}
```

**Comment by JohnnyCrazy** [comment]
Closed via #79
